src/sundials/sundials_hashmap.c

```diff
@@ -320,23 +449,37 @@ SUNErrCode SUNHashMap_Sort(SUNHashMap map, SUNHashMapKeyValue** sorted,
   **Returns:**
     * A SUNErrCode indicating success or a failure
  */
-#if SUNDIALS_MPI_ENABLED
```

Comment:
I think this was needed to avoid an unused function warning because this is only called when MPI is enabled (or at least it was before)

Reply:
I will add `SUNDIALS_MAYBE_UNUSED` to the function instead.